src/nvim/decoration.c

```diff
@@ -702,8 +705,41 @@ int sign_item_cmp(const void *p1, const void *p2)
   return 0;
 }
 
-static const uint32_t sign_filter[4] = {[kMTMetaSignText] = kMTFilterSelect,
-                                        [kMTMetaSignHL] = kMTFilterSelect };
+static const uint32_t conceal_filter[kMTMetaCount] = {[kMTMetaConcealLine] = kMTFilterSelect };
+
+bool decor_conceal_line(win_T *wp, buf_T *buf, int row, bool moved)
```

Comment:
is `moved` for the cursor? perhaps `cursor_moved` is more explicit

Reply:
Yes but the argument only made sense for cursor movement to the next/previous line, so I removed it (and replaced it with separate calls).